bread/layout/components/search_select.py

```diff
@@ -5,6 +5,10 @@
 from .loading import Loading
 from .tag import Tag
 
+ITEM_SELECTOR = "search_result_item"
+ITEM_LABEL_SELECTOR = "search_result_label"
+ITEM_VALUE_SELECTOR = "search_result_value"
+
```

Comment:
Two things about using selectors within components (in general):
1. They should be generic CSS selectors which can directly be passed to ```document.querySelector()``` or ```$()```. Meaning we should not assue it to be a class or an id or anything more specific than a CSS selector (except there is a good reason for it of course).
2. Secondly they should always be parameterizable by the calling code (except they are only used internally, like most cases of ```htmlgenerator.html_id``` and similar). It might be impossible for the using code to change or set a certain CSS class in some cases. Or we have clashes with multiple elements using the same class for some reason. That is also the reason why we have URL query-parameters parameterizable in many places. This kind of parameterize-ability can save the day :fireworks: 

I would even say that we should not provide default selectors. The point is, the code which instantiates the search-component knows the URL and should also know the structure of the returned result. If we provide default selectors in the search_select component we create an unnecessary dependency between the view code and the search_select code, but the dependency should be more between the search_select "user" and the according view.

So, long speech in short: Could you change the variables to be ```__init__``` parameters and allow any CSS selectors?

The calling code in basxconnect could then e.g. use ```"li"```, ```":scope > :last-child"``` and ```":scope > :first-child"``` for the three values (or something nicer, depending on how you structure the view HTML)